docs/docs/constraints.md

```diff
@@ -4,31 +4,31 @@ title: Constraints
 
 # Constraints
 
-Constraints control where apps run to allow optimizing for fault tolerance or locality. They are made up of three parts: a field name, an operator, and an optional parameter. The field can be the slave hostname or any Mesos slave attribute.
+Constraints control where apps run to enable you to optimize for fault tolerance or location. Constraints have three parts: a field name, an operator, and an optional parameter. The field can be the slave hostname or any Mesos slave attribute.
```

Comment:
optimize for fault tolerance: try to spread tasks of an app to as many different hosts as possible.
optimize for location: try to pack as many tasks of an app to the same node.